engine-spring/core/src/main/java/org/operaton/bpm/engine/spring/components/jobexecutor/SpringJobExecutor.java

```diff
@@ -59,10 +60,16 @@ public void executeJobs(List<String> jobIds, ProcessEngineImpl processEngine) {
 	  try {
       taskExecutor.execute(getExecuteJobsRunnable(jobIds, processEngine));
     } catch (RejectedExecutionException e) {
-
       logRejectedExecution(processEngine, jobIds.size());
       rejectedJobsHandler.jobsRejected(jobIds, processEngine, this);
-    }
+		} finally {
+			if (taskExecutor instanceof ThreadPoolTaskExecutor) {
```

Comment:
We have refactored "instanceof" checks to use the "new" style with direct assignment, I'd prefer if we did this in the backports directly, to not import this old style into our repo again